.gitattributes

```diff
@@ -0,0 +1,8 @@
+/docs export-ignore
+/examples export-ignore
+/tests export-ignore
+.gitignore export-ignore
+.gitattributes export-ignore
+.travis.yml export-ignore
+package.xml export-ignore
+phpdoc.sh export-ignore
```

Comment:
I just removed this (old) script from the repository (d64dc053c8513aeae64d95b69911886dd7744ebc) so you can remove it from here.